### PR TITLE
Mobile App Deployment Guide Iteration - Voice and Tone

### PR DESCRIPTION
**Pull request by faase**
#### Summary
Rework to make the guide fit better with the voice and tone guidelines and following the 3Cs

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-24003

### CODE REVIEW

File: source/deployment/mobile-app-deployment.rst [hunk 1]
```diff
@@ -15,67 +15,57 @@ Leaders, Administrators, and/or Champions
     **Note:** Some features described in this guide are only available in Enterprise Edition E10 and/or E20.
 
 *******************
-Learning Objectives
+Learning objectives
 *******************
 
 This guide provides foundational information necessary when developing a plan for a proof of concept and/or enterprise mobile deployment. 
 
-It does not include technical implementation guidance. Links to relevant documentation are included however.
```
Comment (wiersgallak) [inline source/deployment/mobile-app-deployment.rst:23]: Recommend moving to the end of this section and modifying to "Technical implementation guides are included as  links to keep the information in this guide focused."

File: source/install/install-kubernetes-operator.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@ Installing the Operators
 ============================
 
 The required operators are installed using kubectl and each operator is created in its own namespace. Shorter namespace names
-are recommended. You can install and run multiple Mattermost installations in the same cluster using different namespaces.
+are recommended. You can install and run multiple Mattermost installations in the same cluster using different namespaces. Make sure to create operators in unused namespaces only.
```
Comment (jasonblais) [inline source/install/install-kubernetes-operator.rst:7]: Did you intend to make this change here or in another PR?

File: source/deployment/mobile-app-deployment.rst [hunk 3]
```diff
@@ -469,12 +449,12 @@ As part of configuring your EMM solution, you should consider:
...
 
 ****************************************************
-Configuring the MPNS for Your Custom Mattermost Apps
+Configuring the MPNS for your custom Mattermost apps
 ****************************************************
 
 Building and distributing the Mattermost mobile apps requires you to deploy an instance of the MPNS.
 
-As part of the process of building the applications you will need to sign the applications. You must also obtain the appropriate certificate for both Android and iOS. If this is not done, the applications will not be able to interact with you instance of the MPNS.
+As part of the process of building the applications you will need to sign the applications. You must also obtain the appropriate certificate for both Android and iOS. If this is not done, the applications will not be able to interact with Your instance of the MPNS.
```
Comment (jasonblais) [inline source/deployment/mobile-app-deployment.rst:457]:
```suggestion
As part of the process of building the applications you will need to sign the applications. You must also obtain the appropriate certificate for both Android and iOS. If this is not done, the applications will not be able to interact with your instance of the MPNS.
```

File: source/deployment/mobile-app-deployment.rst [hunk 4]
```diff
@@ -15,67 +15,57 @@ Leaders, Administrators, and/or Champions
...
-
-For example, do you want the simplest, fastest deployment possible?
-
-Using the publicly available Mattermost mobile apps on Google Play and Apple App Store can reduce a deployment to minutes.
-
-Better yet, this model can be rolled out to thousands of users in many different countries and languages.
+Our highest priority is making sure you have what you need to be successful, but keep in mind that any change, even beneficial, will have an impact on your users. To ensure your success you need to identify the goals for your mobile deployment.
 
-Of course, large enterprise deployments will take more time, and the overall goals for your deployment will depend on many factors.
+For example, using the publicly available Mattermost mobile apps on Google Play and Apple App Store can reduce a deployment to minutes. Better yet, this model can be rolled out to thousands of users in many different countries and languages.
```
Comment (jasonblais) [inline source/deployment/mobile-app-deployment.rst:62]: We can write more concisely by being direct, without losing the intended tone to the reader

```suggestion
For example, you can deploy to thousands of users across the world in minutes with the publicly available Mattermost mobile apps on Google Play and Apple App Store.
```

File: source/deployment/mobile-app-deployment.rst [hunk 5]
```diff
@@ -15,67 +15,57 @@ Leaders, Administrators, and/or Champions
...
 
-We recommend taking a moment to write down what a successful deployment looks like. To start, ask yourself and your team what you want to achieve.
+Of course, large enterprise deployments will take more time, and the overall goals for your deployment will depend on many factors. We recommend taking a moment to write down what a successful deployment looks like. To start, ask yourself and your team what you want to achieve, what are your desired outcomes?
 
-Put another way, what are your desired outcomes?
+This takes a small amount of time up front. But, we have seen the difference this makes in successful projects.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:66]:
```suggestion
This takes a small amount of time upfront. But, we have seen the difference this makes in successful projects.
```

File: source/deployment/mobile-app-deployment.rst [hunk 6]
```diff
@@ -94,28 +84,26 @@ At the same time, you should begin thinking about technical and security require
...
+While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.
 
 This primer serves as an introduction to Mattermost mobile technologies. It is a complement to our technical documentation, and focuses on three main areas:
 
 - Technical Requirements
 - Basic Features and Technologies
 - Troubleshooting
 
-Regardless of your expertise, we recommend reading this section before starting your deployment.
+Regardless of your expertise, we recommend reading this section before starting your deployment. And keep in mind, we are here to support_ you.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:103]:
```suggestion
Regardless of your expertise, we recommend reading this section before starting your deployment. And keep in mind, we are here to support you.
```

Comment (justinegeffen) [reply source/deployment/mobile-app-deployment.rst:103]: Not sure if this is meant to be a link.

File: source/deployment/mobile-app-deployment.rst [hunk 7]
```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
...
+Again, in the AppConfig Community’s words, "Users benefit with instant mobile productivity and a seamless out-of-the box experience, and businesses benefit with secure work-ready apps with minimal setup required while leveraging existing investments in Enterprise Mobility Management (EMM), VPN, and identity solutions. Put another way, your apps are simpler to configure, secure and deploy."
 
-For now, focus on that last part, “... your apps are simpler to configure, secure and deploy.” AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
+For now, focus on that last part, "... your apps are simpler to configure, secure and deploy." AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:210]: Wondering if the three quotes mentioned above should have links to the article in question? Or are they all from: https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html?

File: source/deployment/mobile-app-deployment.rst [hunk 8]
```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
...
-AppConfig (Supported) vs. App Wrapping (Not Supported)
+AppConfig (supported) vs. app wrapping (not supported)
 ======================================================
 
     **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risks.
 
-So app wrapping isn’t supported, and the AppConfig standard is, but what is the difference between the two?
-
-While there are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining both, let’s take a look at AppConfig first.
+There are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining AppConfig and app wrapping, let us take a look at AppConfig first.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:198]: Should we change this to "a great article" or possibly include a few more articles to round it off?

File: source/deployment/mobile-app-deployment.rst [hunk 9]
```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
...
+For now, focus on that last part, "... your apps are simpler to configure, secure and deploy." AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
 
 Application (App) Wrapping
 --------------------------
-From the article we `referenced earlier <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_, application(app) wrapping involves “...the use of an SDK from an EMM vendor that allows a developer or admin to deploy an API that enables management policies to be set up.” 
+From the article we `referenced earlier <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_, application(app) wrapping involves "...the use of an SDK from an EMM vendor that allows a developer or admin to deploy an API that enables management policies to be set up."
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:213]:
```suggestion
From the article we `referenced earlier <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_, application (app) wrapping involves "...the use of an SDK from an EMM vendor that allows a developer or admin to deploy an API that enables management policies to be set up." 
```

File: source/deployment/mobile-app-deployment.rst [hunk 10]
```diff
@@ -251,19 +231,19 @@ This incompatibility is not an issue with the Mattermost application. It results
 In the end, this sets app wrapping in a negative light. This is the reason the `AppConfig Community <https://www.appconfig.org>`_ came together to create a standard. AppConfig is a modern, efficient, and scalable approach to enterprise mobile management. In that regard, the Mattermost application benefits from being built around this standard.
 
 ******************
-Mobile VPN Options
+Mobile VPN options
 ******************
 
 A Virtual Private Networks (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs. We won't go into detail here.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:237]:
```suggestion
A Virtual Private Network (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs. We won't go into detail here.
```

File: source/deployment/mobile-app-deployment.rst [hunk 11]
```diff
@@ -251,19 +231,19 @@ This incompatibility is not an issue with the Mattermost application. It results
...
+There are VPN options which depend on the requirements of your organization. You should also consider the demands/needs of your users. Regardless, this can impact your approach to mobile deployment.
 
-For the Mattermost mobile application, we’ll discuss two options: a device VPN or per-app VPN.
+For the Mattermost mobile application, we will discuss two options: a device VPN or per-app VPN.
 
     **Note:** We suggest `following our recommended steps <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`_ to secure your deployment.
 
 **Device VPN**
-    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications. Of course even for devices provided by your business, this tends not to be the modern approach.
+    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:246]:
```suggestion
    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issues for personal applications.
```

File: source/deployment/mobile-app-deployment.rst [hunk 12]
```diff
@@ -348,13 +328,13 @@ To understand what’s involved, have your development team `read through our do
...
 #####################################################
 
 To proceed you must have a Mattermost Server installed and accessible. This is also true for using the hosted version of the Mattermost push proxy (HPNS).
 
 *********************************
-Accessing the Mobile Applications
+Accessing the mobile applications
 *********************************
 
 The Mattermost mobile application is available for both Android and iOS devices. At this point, it’s as simple as having your users download the application and `point to your Mattermost Server URL <https://docs.mattermost.com/help/getting-started/signing-in.html#ios-setup>`_.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:340]:
```suggestion
The Mattermost mobile application is available for both Android and iOS devices. At this point, it is as simple as having your users download the application and `point to your Mattermost Server URL <https://docs.mattermost.com/help/getting-started/signing-in.html#ios-setup>`_.
```

File: source/deployment/mobile-app-deployment.rst [hunk 13]
```diff
@@ -384,15 +364,15 @@ When going this route, you should consider:
...
 Configuring your Mattermost Server to use the Mattermost HPNS is a single configuration item. This is covered in our `Hosted Push Notification Service documentation <https://docs.mattermost.com/mobile/mobile-hpns.html>`_. 
 
 Next your users would need to install the mobile application on their device. If desired, you can further configure security capabilities using an EMM provider.
 
 **********************************
-Updating via the Public Store Apps
+Updating via the public store apps
 **********************************
 
 While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost mobile applications are available. Simultaneously, you should check any compatibility requirements for the mobile apps and the Mattermost server.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:378]:
```suggestion
While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost mobile applications are available. Simultaneously, you should check any compatibility requirements for the mobile apps and the Mattermost Server. 
```

File: source/deployment/mobile-app-deployment.rst [hunk 14]
```diff
@@ -413,16 +393,16 @@ Consult the `Mattermost mobile app changelog <https://github.com/mattermost/matt
...
 
 ###################################
-Mobile Deployment via Custom Builds
+Mobile deployment via custom builds
 ###################################
 
-Choosing this model means you’ve decided not to use the mobile applications Mattermost has made available via the public app stores. 
+Choosing this model means you have decided not to use the mobile applications Mattermost has made available via the public app stores. 
 
 This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost server.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:401]:
```suggestion
This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost Server.
```

File: source/deployment/mobile-app-deployment.rst [hunk 15]
```diff
@@ -413,16 +393,16 @@ Consult the `Mattermost mobile app changelog <https://github.com/mattermost/matt
...
 
 Finally, this choice also means you cannot use the hosted version of the Mattermost Push Notification Service (MPNS).
 
-A great first step for this deployment model involves `reviewing the documentation <https://developers.mattermost.com/contribute/mobile/>`_. Your development team should run through the requirements for building the applications. This will help understand the overall impact when choosing this option. This documentation provides guidance on building, compiling, and signing. It also includes information for customizing the apps.
+As a first step for this deployment model is `reviewing the documentation <https://developers.mattermost.com/contribute/mobile/>`_. Your development team should run through the requirements for building the applications. This documentation provides guidance on building, compiling, and signing. It also includes information for customizing the apps.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:405]:
```suggestion
A first step for this deployment model is `reviewing the documentation <https://developers.mattermost.com/contribute/mobile/>`_. Your development team should run through the requirements for building the applications. This documentation provides guidance on building, compiling, and signing. It also includes information for customizing the apps.
```

File: source/deployment/mobile-app-deployment.rst [hunk 16]
```diff
@@ -448,18 +428,18 @@ Option 2 - Apple App Store and Google Play (less common):
 - This process is more common if you are looking to white label the app to remove Mattermost branding
 
 ******************************************************
-Using an EMM Provider with Your Custom Mattermost Apps
+Using an EMM Provider with your custom Mattermost apps
 ******************************************************
 
 If you will be using an EMM provider, note that Mattermost does not support App Wrapping. Instead we use the AppConfig standard.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:434]:
```suggestion
If you will be using an EMM provider, note that Mattermost does not support app wrapping. Instead we use the AppConfig standard.
```

File: source/deployment/mobile-app-deployment.rst [hunk 17]
```diff
@@ -448,18 +428,18 @@ Option 2 - Apple App Store and Google Play (less common):
...
 
 In some instances, there is an incompatibility with App Wrapping and React Native Applications. React Native is the technology used to develop the Mattermost mobile applications. The Mattermost mobile app will not function properly when using app wrapping. In fact, a majority of the app wrappers EMMs provide do not support WebSocket connections.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:436]:
```suggestion
In some instances, there is an incompatibility with app wrapping and React Native applications. React Native is the technology used to develop the Mattermost mobile applications. The Mattermost mobile app will not function properly when using app wrapping. In fact, a majority of the app wrappers EMMs provide do not support WebSocket connections.
```

File: source/deployment/mobile-app-deployment.rst [hunk 18]
```diff
@@ -485,7 +465,7 @@ The documentation topics listed below guide installation and configuration for y
 - `Additional FAQs <https://docs.mattermost.com/mobile/mobile-faq.html#>`_
 
 ************************************
-Updating Your Custom Mattermost Apps
+Updating your custom Mattermost apps
 ************************************
 
 While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost Mobile applications are available. We highly recommend you update your custom Mattermost apps for any security or service releases. At the same time, if you have updated the apps, prior to distribution, check any compatibility requirements for the mobile apps and the Mattermost server.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:471]:
```suggestion
While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost mobile applications are available. We highly recommend you update your custom Mattermost apps for any security or service releases. At the same time, if you have updated the apps, prior to distribution, check any compatibility requirements for the mobile apps and the Mattermost server. 
```

File: source/install/install-kubernetes-operator.rst (hunk 2 above)
Comment (faase) [reply source/install/install-kubernetes-operator.rst:7]: no, this belongs to a different PR. Let me find out how this could happen.

File: source/deployment/mobile-app-deployment.rst (hunk 6 above)
Comment (faase) [reply source/deployment/mobile-app-deployment.rst:103]: it is ;)

File: source/deployment/mobile-app-deployment.rst (hunk 7 above)
Comment (faase) [reply source/deployment/mobile-app-deployment.rst:210]: That is a great question! AFAIK they are all direct quotes from [appconfig.org](https://www.appconfig.org)

File: source/deployment/mobile-app-deployment.rst (hunk 8 above)
Comment (faase) [reply source/deployment/mobile-app-deployment.rst:198]: I'd change it to 

> Here is a great article ...

WDYT @justinegeffen

Comment (justinegeffen) [reply source/deployment/mobile-app-deployment.rst:198]: I'd possibly reword it to "Here is a helpful article" as opposed to "great" unless we're certain that the referenced article is considered great by multiple trusted sources. 3/5 as this type of writing is different to technical documentation.

File: source/deployment/mobile-app-deployment.rst (hunk 7 above)
Comment (justinegeffen) [reply source/deployment/mobile-app-deployment.rst:210]: Might be helpful to link to that article or specific section in the article to provide content for the reader? 3/5; not sure what the standard for quotes is in CE docs. :)

File: source/install/install-kubernetes-operator.rst (hunk 2 above)
Comment (faase) [reply source/install/install-kubernetes-operator.rst]: think I solved it

File: source/deployment/mobile-app-deployment.rst (hunk 7 above)
Comment (faase) [reply source/deployment/mobile-app-deployment.rst:210]: No, I mean it's literally on appconfig.com, right on their front page.

appconfig

File: source/deployment/mobile-app-deployment.rst (hunk 8 above)
Comment (faase) [reply source/deployment/mobile-app-deployment.rst]: I'm 0/5 here, I just went with the wording in your suggestion ;)

**Comment by justinegeffen** [comment]
General comment:
I understand the reason behind not using contractions in the document. Wondering if that could be reconsidered so the tone is less formal, and more conversational. It would also align it with the technical docs. I realize the audience is different in this case but the voice/tone may benefit from the consistency.

**Comment by faase** [comment]
> General comment:
> I understand the reason behind not using contractions in the document. Wondering if that could be reconsidered so the tone is less formal, and more conversational. It would also align it with the technical docs. I realize the audience is different in this case but the voice/tone may benefit from the consistency.

Sure, let's discuss this with all involved people in a meeting maybe?

File: source/deployment/mobile-app-deployment.rst [hunk 19]
```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
...
+Again, in the AppConfig Community’s words, "Users benefit with instant mobile productivity and a seamless out-of-the box experience, and businesses benefit with secure work-ready apps with minimal setup required while leveraging existing investments in Enterprise Mobility Management (EMM), VPN, and identity solutions. Put another way, your apps are simpler to configure, secure and deploy."
 
-For now, focus on that last part, “... your apps are simpler to configure, secure and deploy.” AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
+For now, focus on that last part, "... your apps are simpler to configure, secure and deploy." AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:209]:
```suggestion
For now, focus on that last part, "... your apps are simpler to configure, secure and deploy." AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it is our only supported approach.
```

File: source/deployment/mobile-app-deployment.rst [hunk 20]
```diff
@@ -251,19 +231,19 @@ This incompatibility is not an issue with the Mattermost application. It results
 In the end, this sets app wrapping in a negative light. This is the reason the `AppConfig Community <https://www.appconfig.org>`_ came together to create a standard. AppConfig is a modern, efficient, and scalable approach to enterprise mobile management. In that regard, the Mattermost application benefits from being built around this standard.
 
 ******************
-Mobile VPN Options
+Mobile VPN options
 ******************
 
-A Virtual Private Networks (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs. We won't go into detail here.
+A Virtual Private Network (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs. We won't go into detail here.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:237]:
```suggestion
A Virtual Private Network (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs. We will not go into detail here.
```

File: source/deployment/mobile-app-deployment.rst [hunk 21]
```diff
@@ -320,7 +300,7 @@ The mobile applications provided by Mattermost work with our hosted version of t
 *Disadvantages:* No app customization
 
 *********************************************
-Distributing Custom Builds of the Mobile Apps
+Distributing custom builds of the mobile apps
 *********************************************
 
 **Desired Outcomes:**
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:306]:
```suggestion
**Desired Outcomes**:
```

File: source/deployment/mobile-app-deployment.rst [hunk 22]
```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
 
 AppConfig is newer, more modern approach when compared to the previous standard, App Wrapping.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:191]:
```suggestion
AppConfig is newer, more modern approach when compared to the previous standard, app wrapping.
```

File: source/deployment/mobile-app-deployment.rst [hunk 23]
```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
...
 
-We’ll say it several times throughout this guide, and in the next section. If you are using an EMM provider, the only supported path is via AppConfig.
-
-While app wrapping has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
-
-AppConfig (Supported) vs. App Wrapping (Not Supported)
+AppConfig (supported) vs. app wrapping (not supported)
 ======================================================
 
     **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risks.
```
Comment (justinegeffen) [inline source/deployment/mobile-app-deployment.rst:196]:
```suggestion
    **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risk.
```

File: source/deployment/mobile-app-deployment.rst (hunk 7 above)
Comment (justinegeffen) [reply source/deployment/mobile-app-deployment.rst:210]: Great. :)

**Comment by justinegeffen** [comment]
General observation: Not all the instances of contractions have been changed to the full version. Recommend reviewing the doc for this. Happy to give a second pass if needed.

File: source/deployment/mobile-app-deployment.rst [hunk 24]
```diff
@@ -15,67 +15,57 @@ Leaders, Administrators, and/or Champions
...
-By the end of this guide you should have an understanding of the high-level requirements for a successful Mattermost mobile deployment. 
+By the end of this guide you will have an understanding of the high-level requirements for a successful Mattermost mobile deployment. 
 
 This includes:
 
 - Choosing how to obtain and distribute the Mattermost mobile applications
 - Determining the ideal mobile deployment model for your organization
 - Understanding the resource requirements when choosing to build your own Mattermost mobile apps
 
+Technical implementation guides are included as links to keep the information in this guide focused.
```
Comment (jasonblais) [inline source/deployment/mobile-app-deployment.rst:31]:
```suggestion
Technical implementation guides are included as links to keep the information in this guide concise.
```

Comment (jasonblais) [reply source/deployment/mobile-app-deployment.rst:31]: ('focused' may have multiple meanings such as 'focused on one topic')

**Comment by jasonblais** [comment]
^this is a good iteration in itself, so we could review for conciseness in a follow-up PR. I realize this has had some conversations already, and don't want to block improvements already submitted.

**Comment by justinegeffen** [comment]
Following on from @jasonblais's comment, my two comments around contractions and formatting should also be considered non-blocking/applicable to the next iteration.

**Comment by faase** [comment]
> Beyond the one suggestion that I offered, were other parts reviewed for conciseness?
> 
> [This can be in a separate PR] Similar to the Enterprise deployment guide, would we want to more clearly define the audience, e.g. `Leaders, Administrators, and/or Champions responsible for deploying Mattermost mobile apps in their organization.`

@jasonblais I most certainly tried to remove or rephrase everything that wouldn't add information or motivate the audience. But than I am still learning to write  concisely.

File: source/deployment/mobile-app-deployment.rst (hunk 6 above)
Comment (wiersgallak) [inline source/deployment/mobile-app-deployment.rst:103]:
```suggestion
Regardless of your expertise, we recommend reading this section before starting your deployment. And keep in mind, we are here to support you.
```

File: source/deployment/mobile-app-deployment.rst [hunk 25]
```diff
@@ -202,35 +188,29 @@ Most large enterprise teams are familiar with `Enterprise Mobile Management <htt
...
+AppConfig (supported) vs. app wrapping (not supported)
 ======================================================
 
-    **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risks.
-
-So app wrapping isn’t supported, and the AppConfig standard is, but what is the difference between the two?
+    **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risk.
 
-While there are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining both, let’s take a look at AppConfig first.
+Here is `a helpful article  <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining AppConfig and app wrapping, let us take a look at AppConfig first.
```
Comment (wiersgallak) [inline source/deployment/mobile-app-deployment.rst:198]:
```suggestion
Here is `a helpful article  <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining AppConfig and app wrapping. Let us take a look at AppConfig first.
```

File: source/deployment/mobile-app-deployment.rst (hunk 6 above)
Comment (faase) [reply source/deployment/mobile-app-deployment.rst:103]: This is on purpose and should link (at least it works on my local build) to the support section of the document while rendering as `support`.

Comment (justinegeffen) [reply source/deployment/mobile-app-deployment.rst:103]: Non-blocking comment: I'm wondering if a link might render better - i.e., as a hyperlink and not the same font that we use for indicating file names (e.g., config.json).

Comment (justinegeffen) [reply source/deployment/mobile-app-deployment.rst:103]: Disregard my comment. I've done a local build and the link renders as expected. 

Screen Shot 2020-06-23 at 12 41 30
